### PR TITLE
verbose flag added for k2cli 

### PR DESCRIPTION
**Pull request by DStorck**
user can now add -v=true to stream logs ( same output as running `up` in k2 ).

### CODE REVIEW

File: cmd/container_helpers.go [hunk 1]
```diff
@@ -43,6 +44,29 @@ func base64EncodeAuth(auth types.AuthConfig) (string, error) {
...
+			resp.ID,
+			types.ContainerLogsOptions{
+				ShowStdout: true,
+				Follow: true,
+			})
+    if err != nil {
+        log.Fatal(err)
+    }
+
+		defer reader.Close()
```
Comment (davidewatson) [inline cmd/container_helpers.go]: You can use go fmt to fix up indentation to conform to golang's style guidelines.

**Comment by davidewatson** [comment]
LGTM.

File: cmd/container_helpers.go (hunk 1 above)
Comment (DStorck) [inline cmd/container_helpers.go]: I have no idea why it looks indented in this PR, i ran `go fmt` before submitting this and it didn't fit it .

Comment (davidewatson) [inline cmd/container_helpers.go]: I don't know why it wasn't fixed, but the difference here is tabs vs. spaces.

**Comment by leahnp** [comment]
So you pass ` -v=true` when running k2cli up? LGTM

**Pull request merged**